### ISSUE TITLE
EDA hard to follow

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_GROUP_312#28]
Some areas to improve on as you adapt this in your pipeline:

- EDA initial thoughts (Step 4) don't seem connected to the presented numbers
- Would be useful to include figures in the proposal when describing main EDA results - or for purposes of Milestone 2, put more description in the EDA. It is confusing to have to flip between the two. 
- Could be more efficient / readable when producing min/max/mean summaries (make a table - use `describe()`)
- What are the "lines" at house value 500,000 in your Step 7?

**Comment by slbsolomon** [comment]
Hi Kate,

#39 addressed this issue.

The "lines" at house value 500000 we believe is due to a cut-off of the data. We unfortunately couldn't find information about this from the dataset information, but we have reason to believe that any value larger than 500000 for median housing value was assigned 500000. It appears that a similar approach was used for median income with a cutoff of 150000.
